src/Makefile.am

```diff
@@ -238,6 +238,7 @@ BITCOIN_CORE_H = \
   util/rbf.h \
   util/ref.h \
   util/settings.h \
+  util/sock.h \
```

Comment:
Please create new general utilities like this in `src/compat`, `src/support` or `src/util`, not in the source root.

Reply:
Hmm, the newly added `sock.h` is in `src/util`, so that's already the case, no?

Reply:
Huh. Oops. I misread this. Seems good then.